.changeset/thick-lions-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/auth': minor
```

Comment:
Can you add `"firebase": minor` as well? Sorry, we have to manually do this every time we have a bump greater than patch, changesets isn't able to automatically do it.

Reply:
Done.